client/src/components/layouts/variables.css

```diff
@@ -15,7 +15,9 @@
   --yellow-light: #ffc300;
   --yellow-dark: #4d3800;
   --blue-light: rgb(153, 201, 255);
+  --blue-light-translucent: rgb(153, 201, 255, 0.3);
```

Comment:
@ojeytonwilliams rgba? :)

Reply:
🤦🏽‍♂️ - nice catch.

Reply:
rgba is just an alias for rgb in modern browsers. I think we're fine.

Reply:
Perfect, learnt something new today.